plugins/woocommerce/package.json

```diff
@@ -68,12 +68,14 @@
 		"allure-commandline": "^2.17.2",
 		"allure-playwright": "^2.0.0-beta.16",
 		"autoprefixer": "9.8.6",
+		"axios": "^0.24.0",
```

Comment:
I added `axios` as a dependency here instead of `request` (which is what the utility functions in `e2e-environment` use) because I found the syntax in `axios` to be simpler. Also, `request` is already deprecated.